docs-content/upgrading.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Updating CyberArk Conjur for PCF
+owner: Partners
+---
+
+This topic explains how to upgrade the CyberArk Conjur for Pivotal Cloud Foundry (PCF) tile.
+
+## <a id="upgrade"></a> Upgrade CyberArk Conjur for PCF
+
+Upgrading the CyberArk Conjur for PCF tile generally requires the same steps in the Ops Manager
+as when you first install the tile. Your configuration settings will migrate to the new version
+when you add the upgrade tile to the Ops Manager.
+
+1. Review the Release Notes for the version you are upgrading to, also check for any [version specific
+   upgrade instructions below](#version-notes).
+
+2. Download the desired tile version from Pivotal Network.
+
+3. Open the Ops Manager Dashboard and select **Import a Product** to upload the new tile file.
+
+4. Under the **Import a Product** button, click the **+** next to **CyberArk Conjur Service 
+   Broker for PCF**. This adds the updated tile to your Ops Manager staging area.
+
+5. Click the **CyberArk Conjur Service Broker for PCF** tile to review the configuration settings. Click **Save**
+   on any tabs where you modify the settings.
+
+6. Open the Ops Manager Dashboard and click **Review Pending Changes** and **Apply Changes**.
+
+## <a id="version-notes"></a> Version Specific Upgrade Instructions
+
+### From 0.3.3 to 1.0.0
+
+Before performing the upgrade from tile version 0.3.3 to 1.0.0 using the Ops Manager, you must first
+rename the existing service broker and service broker application in PCF.
+
+Version 1.0.0 also updates the Conjur Buildpack to version 2.0.0. This version changes the kind of
+buildpack from a decorator buildpack to a supply buildpack. This will likely require updating how
+your applications include and use the buildpack. More information about using the CyberArkConjur Buildpack
```

Comment:
`CyberArkConjur` => `Conjur`?

Reply:
To clarify, you want CyberArk removed, or is this just calling out the missing space?